plugins/drop_folder/lib/kDropFolderEventsConsumer.php

```diff
@@ -266,6 +266,10 @@ private function onEntryStatusChanged( $entry )
 	{
 		// Handle only files that are still in the PROCESSING state, which were left
 		// in this state due to AUTO_DELETE_WHEN_ENTRY_IS_READY delete policy.
+        if ($entry->getIsTemporary())
+        {
+            $entry = $entry->getReplacedEntryId();
+        }
 		$dropFolderFiles = DropFolderFilePeer::retrieveByEntryIdPartnerIdAndStatuses($entry->getId(), $entry->getPartnerId(), array(DropFolderFileStatus::PROCESSING));
```

Comment:
Need to modify $entry->getId to '$entryId' (based on the above fix suggestion)
$dropFolderFiles = DropFolderFilePeer::retrieveByEntryIdPartnerIdAndStatuses($entryId, $entry->getPartnerId(), array(DropFolderFileStatus::PROCESSING));

+ need to further inspect the rest of this function to see if other alignments should be made